backend/src/routes/tenant.py

```
@@ -1,11 +1,13 @@
 from flask import Blueprint, request, jsonify
+from flask_jwt_extended import jwt_required
 from src.models.database import db, Tenant, User
 from werkzeug.security import generate_password_hash
 import uuid
```

Ruff (0.11.9)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)